cmd/nerdctl/container_run_network_linux_test.go

```diff
@@ -575,3 +575,73 @@ func TestRunContainerWithMACAddress(t *testing.T) {
 		}
 	}
 }
+
+func TestRunContainerWithStaticIP6(t *testing.T) {
```

Comment:
What about non-static?

Reply:
It was my mistake to name it `static`. In the test table, an item contains an empty IP string that can be assigned a non-static IP6. 

https://github.com/containerd/nerdctl/blob/f29f5f538ca468397af5d0aeeb5d233e6ac98324/cmd/nerdctl/container_run_network_linux_test.go#L507-L511

Later, it is verified whether it is a valid IP6 address.

https://github.com/containerd/nerdctl/blob/f29f5f538ca468397af5d0aeeb5d233e6ac98324/cmd/nerdctl/container_run_network_linux_test.go#L545-L547